src/main/java/com/checkscanner/check/DataLoader.java

```diff
@@ -19,7 +19,7 @@
 	public void loadSomeData() {
 		final File dir = new File("src/main/resources/venues/");
 		final File[] directoryListing = dir.listFiles();
-		final int count = 0;
+		int count = 0;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "count" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1481)